From 4164cd68d45ec4b3b46c22ed981c81f766154428 Mon Sep 17 00:00:00 2001
From: Vadim Zeitlin <vadim@wxwidgets.org>
Date: Thu, 17 Jun 2010 22:30:27 +0000
Subject: Backport fix from bakefile svn for autoconf 2.65.

Use AC_REQUIRE() instead of directly using a macro, the latter doesn't work
with autoconf 2.65.

This is a backport of r1337 from bakefile svn to allow people to use autoconf
2.65 immediately and will become unnecessary when we update to the next
bakefile version.

git-svn-id: https://svn.wxwidgets.org/svn/wx/wxWidgets/trunk@64620 c3d73ce0-8a6f-49c7-b76d-6d57e0e08775

diff --git a/build/aclocal/bakefile-lang.m4 b/build/aclocal/bakefile-lang.m4
index 4cc4a67..ec0f711 100644
--- a/build/aclocal/bakefile-lang.m4
+++ b/build/aclocal/bakefile-lang.m4
@@ -240,7 +240,7 @@ dnl check for different proprietary compilers depending on target platform
 dnl _AC_BAKEFILE_PROG_COMPILER(LANG)
 AC_DEFUN([_AC_BAKEFILE_PROG_COMPILER],
 [
-    AC_PROG_$1
+    AC_REQUIRE([AC_PROG_$1])
 
     dnl Intel compiler can be used under several different OS and even
     dnl different architectures (x86, amd64 and Itanium) so it's easier to just
